### PR TITLE
Linux build for CI of other packages

### PR DESCRIPTION
**Pull request by angeloskath**
This PR adds the option to build a linux package compatible with Ubuntu >= 20.04 for use with CI of other packages.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -123,8 +123,8 @@ else()
     /usr/include
     /usr/local/include
     $ENV{BLAS_HOME}/include)
-  message(STATUS "Blas lib" ${BLAS_LIBRARIES})
-  message(STATUS "Blas include" ${BLAS_INCLUDE_DIRS})
+  message(STATUS "Blas lib " ${BLAS_LIBRARIES})
+  message(STATUS "Blas include " ${BLAS_INCLUDE_DIRS})
```
Comment (awni) [inline CMakeLists.txt:127]: 😄  thanks

File: .circleci/config.yml [hunk 2]
```diff
@@ -170,12 +173,61 @@ jobs:
...
+            PYTHON=python<< parameters.python_version >>
+            apt-get update
+            apt-get upgrade -y
+            DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata
+            apt-get install -y apt-utils
+            apt-get install -y software-properties-common
+            add-apt-repository -y ppa:deadsnakes/ppa
+            apt-get install -y $PYTHON $PYTHON-dev $PYTHON-full
+            apt-get install -y libblas-dev liblapack-dev liblapacke-dev
+            apt-get install -y build-essential git
```
Comment (awni) [inline .circleci/config.yml:200]: 🙈

**Review by awni — APPROVED** [review]
LFG, thank you !!!!

File: .circleci/config.yml (hunk 2 above)
Comment (angeloskath) [reply .circleci/config.yml:200]: Haha yeah indeed. That's about 3-4 hours of banging my head against the wall trying to make `apt-get` not ask me the timezone ...

Comment (angeloskath) [reply .circleci/config.yml:200]: Another 3-4 were realizing that I have to choose the platform for `auditwheel`. I feel extremely stupid about this but everybody builds on a manylinux docker and apparently you don't specify the platform there or people just omit it from all the docs that are on the internet.

**Pull request merged**